google/cloud/sql/connector/ephemeral.py

```diff
@@ -0,0 +1,91 @@
+"""
```

Comment:
nit: These functions probably don't each need their own file, they could probably be a single file (called `refresh` or `refresh_utils` maybe?)